packages/core/tests/proofs.test.ts

```diff
@@ -365,8 +371,162 @@ describe('Present Proof', () => {
         requestedAttributes: attributes,
         requestedPredicates: predicates,
       })
-    ).rejects.toThrowError(
-      `The proof request contains an attribute group name that matches a predicate group name: age`
-    )
+    ).rejects.toThrowError(`The proof request contains duplicate items: age`)
+  })
+
+  test('attribute names match, same cred def filter', async () => {
```

Comment:
These tests are great! I think it would be good to move them to a util test in the utils __tests__ directory. Then add tests to this file that makes sure the method is called where it should be (createRequest, etc...)